recipe/build.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+# from Continuum's build.sh
+
+LIB=$PREFIX/lib
+mkdir $LIB
+cd $LIB
+
+if [ $ARCH == '64' ]; then
+    cp /usr/local/lib64/libgfortran.so.3.0.0 .
```

Comment:
so that file doesn't exist on our docker build image.

I don't fully understand.

do i:
1) use /usr/lib64/libgfortran.so.3.0.0
2) use /opt/rh/devtoolset-2/root/usr/lib/gcc/x86_64-redhat-linux/4.8.2/libgfortran.so [this is from devtoolset-2-gcc-gfortan rpm package, which other things in conda-forge are relying upon for gfortran compiling I think already
3) none of the above, due to whatever you're working on

Cheers,